crates/holochain_cascade/src/lib.rs

```diff
@@ -223,7 +226,145 @@ impl Cascade<HolochainP2pDna> {
     }
 }
 
-impl<Network> Cascade<Network>
+/// TODO
+#[async_trait::async_trait]
+#[mockall::automock]
+pub trait Cascade {
+    /// Retrieve [`Entry`] from either locally or from an authority.
```

Comment:
```suggestion
    /// Retrieve [`Entry`] either locally or from an authority.
```

Reply:
Maybe that only sounds funny to me, it's used like this in the following comments too.